### ISSUE TITLE
How to query without a schema?

### ISSUE THREAD

**Issue by rathboma** [ashkan18/graphlient#82]
Hey!

Thanks for making a gem.

I don't have a schema for my endpoint, but I do have a query + params. Is there a way to query without a schema (file or http)?

Thanks!

**Comment by dblock** [comment]
Not sure, but this might be helpful: https://github.com/rmosolgo/graphql-ruby/pull/2327 and https://github.com/github/graphql-client/issues/211 (another client)

**Comment by yuki24** [comment]
Yes, @dblock is right about this. Theoretically you may not need a schema in production, but this needs to be implemented in the underlaying gem first.
